spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Admin::Reports::RegisteredAdviserController, type: :controller do
+  describe '#filename' do
```

Comment:
I discussed this with @margOnline and she's happy to keep the tests where they are and as a private method in the controller where it originates from.